sqlite3_opt_serialize_test.go

```diff
@@ -0,0 +1,120 @@
+// +build !libsqlite3 sqlite_serialize
+
+package sqlite3
+
+import (
+	"database/sql"
+	"database/sql/driver"
+	"os"
+	"testing"
+)
+
+func TestSerialize(t *testing.T) {
+	d := SQLiteDriver{}
+
+	srcConn, err := d.Open(":memory:")
```

Comment:
If you just use `:memory:` you can run into weird issues due to Go's connection pooling. Please use file:/TestSerialize?vfs=memdb` instead.